tests/schemas/api_v3/block.schema.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Joi from 'joi';
+
+const blockSchema = {
+	id: Joi.string().required(),
+	height: Joi.number().integer().min(0).required(),
+	version: Joi.number().required(),
+	timestamp: Joi.number().integer().min(1).required(),
+	generatorAddress: Joi.string().optional(),
```

Comment:
Should this not be required? However, for the snapshot (re-genesis) block, it should be an empty string. Please cross-verify this with the LIP.

```suggestion
	generatorAddress: Joi.string().required(),
```